Algorithm/QCAlgorithm.Framework.Python.cs

```diff
@@ -28,6 +28,7 @@ public partial class QCAlgorithm
         /// Sets the alpha model
         /// </summary>
         /// <param name="alpha">Model that generates alpha</param>
+        [DocumentationAttribute("Algorithm Framework")]
```

Comment:
Cool!
Maybe these could be fined in a single documentation file in common and used like this, avoid any typos
```chsarp

        public const string AlgorithmFramework = "Algorithm Framework";

        [Documentation(AlgorithmFramework)]
```

Reply:
Added by [4615af3](https://github.com/QuantConnect/Lean/pull/6098/commits/4615af38f5d78fe57fe210b64989e8966297693d)